pom.xml

```diff
@@ -117,6 +118,18 @@
       <artifactId>jnr-unixsocket-nodep</artifactId>
       <version>0.3.1</version>
     </dependency>
+    <dependency>
+      <groupId>com.google.code.findbugs</groupId>
+      <artifactId>annotations</artifactId>
+      <version>3.0.0</version>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins.workflow</groupId>
+      <artifactId>workflow-step-api</artifactId>
+      <version>${workflow-jenkins-plugin.version}</version>
```

Comment:
No problem @recena see also the question on dev list that triggered this discussion just in case https://groups.google.com/forum/#!topic/jenkinsci-dev/K7pf_W8NgqA